acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
Not having `facter` in the $LOAD_PATH on Windows sounds like a bug, not something we want to skirt around in the test.